docs/README.md

```diff
@@ -26,24 +26,23 @@ There are three properties to configure the CLI with:
 
 1. **API host** (name or IP address) for the OpenWhisk deployment you want to use.
 2. **Authorization key** (username and password) which grants you access to the OpenWhisk API.
-3. **Namespace** where your OpenWhisk assets are stored.
 
 The CLI will usually have an API host already set. You can check its value with
 `wsk property get --apihost`.
 
-If you know your authorization key and namespace, you can configure the CLI to use them. Otherwise
-you will need to provide one or both for most CLI operations.
+If you know your authorization key, you can configure the CLI to use it. Otherwise,
+you will need to obtain an authorization key for most CLI operations.
```

Comment:
we should link here to how you obtain it - link to wskadmin user create doc? @csantanapr thoughts?